example-compose-files/sq-dce-custom-zip-postgres/docker-compose.yml

```diff
@@ -0,0 +1,228 @@
+services:
+  sonarqube-1:
+    ports:
+      - "9001:9000"
+    healthcheck:
+      test: wget --no-proxy -qO- http://localhost:9000/api/system/status | grep -q -e '"status":"UP"' -e '"status":"DB_MIGRATION_NEEDED"' -e '"status":"DB_MIGRATION_RUNNING"'
+      interval: 25s
+      timeout: 1s
+      retries: 3
+      start_period: 55s
+    build:
+      context: .
+      dockerfile: Dockerfile-app
+      args:
+        SONARQUBE_VERSION: ${SONARQUBE_VERSION}
+    read_only: true
+    hostname: "sonarqube-1"
+    depends_on:
+      search-1:
+        condition: service_healthy
+      search-2:
+        condition: service_healthy
+      db:
+        condition: service_healthy
+    networks:
+      dual:
+        ipv4_address: 192.168.3.91
+        ipv6_address: 2001:db8:3::91
+    cpus: 0.5
+    mem_limit: 4096M
+    mem_reservation: 4096M
+    environment:
+      SONAR_JDBC_URL: jdbc:postgresql://db:5432/sonar
+      SONAR_JDBC_USERNAME: sonar
+      SONAR_JDBC_PASSWORD: sonar
+      SONAR_WEB_PORT: 9000
+      SONAR_CLUSTER_SEARCH_HOSTS: "search-1,search-2"
+      SONAR_CLUSTER_NODE_NAME: "sonarqube-1"
+      SONAR_CLUSTER_HOSTS: "[2001:db8:3::91],[2001:db8:3::92]"
+      SONAR_AUTH_JWTBASE64HS256SECRET: "dZ0EB0KxnF++nr5+4vfTCaun/eWbv6gOoXodiAMqcFo="
+      JAVA_TOOL_OPTIONS: '-Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true'
+      SONAR_WEB_JAVAADDITIONALOPTS: '-Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true'
```

Comment:
Do we want to be this much IPv6 Opinionated? 
I think if that is the case we should make that more clear in the readme.

Reply:
The example is meant for IPv6 installations only...we have other examples, that are more general purpose. Also, @jCOTINEAU removed the explicit IPs in the example...